packages/appeals-service-api/src/lib/notify.js

```diff
@@ -236,6 +236,38 @@ const sendFailureToUploadToHorizonEmail = async (appealId) => {
 	}
 };
 
+const sendLPADashboardInviteEmail = async (user) => {
+	try {
+		const lpa = await lpaService.getLpaByCode(user.lpaCode);
+
+		const recipientEmail = user.email;
+		let variables = {
+			'local planning authority': lpa.getName(),
+			'lpa-link': `${config.apps.appeals.baseUrl}/manage-appeals/your-email-address`
+		};
+
+		const reference = user._id;
+
+		logger.debug(
+			{ recipientEmail, variables, reference },
+			'Sending LPA dashboard invitation email'
+		);
+
+		await NotifyBuilder.reset()
+			.setTemplateId(templates.LPA_DASHBOARD.lpaDashboardInviteEmail)
+			.setDestinationEmailAddress(recipientEmail)
+			.setTemplateVariablesFromObject(variables)
+			.setReference(reference)
+			.sendEmail(
+				config.services.notify.baseUrl,
+				config.services.notify.serviceId,
+				config.services.notify.apiKey
+			);
+	} catch (err) {
+		logger.error({ err, userId: user._id }, 'Unable to send LPA dashboard invitation email');
```

Comment:
Do you really want to swallow the error here?

Reply:
Yeah if we're triggering this from some downstream client it would be good to pass the exception back up the call stack if there is an opportunity to give an end user or whatever triggers this a chance to replay it or at least tell them it's failed. Got to be honest I haven't looked at the notify stuff at all so that might not be relevant in this instance.
Also, again with the caveat of not knowing how the notify sdk works, so please feel free to ignore this, but from line 256 onwards seems like the kind of thing you'd usually look to bootstrap once and not have to worry about each time you called it.

Reply:
the other notify calls are done within a try/catch, so continued that here.

can look at refactoring the await NotifyBuilder calls - there are quite a few of them in the notify.js file